sdk/core/azure-core-serializer-json-jackson/src/main/java/com/azure/core/serializer/json/jackson/JacksonJsonSerializerBuilder.java

```diff
@@ -6,20 +6,20 @@
 import com.fasterxml.jackson.databind.ObjectMapper;
 
 /**
- * Fluent builder class that configures and instantiates instances of {@link JacksonJsonSerializer}.
+ * Fluent builder class that configures and instantiates instances of {@link JacksonSearchSerializer}.
  */
 public final class JacksonJsonSerializerBuilder {
     private ObjectMapper objectMapper;
 
     /**
-     * Constructs a new instance of {@link JacksonJsonSerializer} with the configurations set in this builder.
+     * Constructs a new instance of {@link JacksonSearchSerializer} with the configurations set in this builder.
      *
-     * @return A new instance of {@link JacksonJsonSerializer}.
+     * @return A new instance of {@link JacksonSearchSerializer}.
      */
-    public JacksonJsonSerializer build() {
+    public JacksonSearchSerializer build() {
```

Comment:
Why was the Azure Core implementation renamed? Just prototyping?

Reply:
Fixed. Caused by IDE refactoring